process-link/src/main/kotlin/com/ritense/processlink/service/ProcessLinkActivityService.kt

```diff
@@ -61,10 +61,14 @@ class ProcessLinkActivityService(
     fun openTask(
         @LoggableResource(resourceType = CamundaTask::class) taskId: UUID
     ): ProcessLinkActivityResult<*> {
-        val task = taskService.findTaskOrThrow(
-            byId(taskId.toString())
-                .and(byActive())
-        )
+        val task = try {
+            taskService.findTaskOrThrow(
+                byId(taskId.toString())
+                    .and(byActive())
+            )
+        } catch (exception: Exception) {
```

Comment:
Is this type correct? This would mean that any possible error (which admittedly i can't think of many) would be translated to process link not found.

Reply:
Fixed it based on our discussion